types/wavesurfer.js/index.d.ts

```diff
@@ -75,7 +78,7 @@ declare class WaveSurfer extends Observer {
     destroyPlugin(name: string): WaveSurfer;
     empty(): void;
     exportImage(format?: string, quality?: number, type?: "dataURL" | "blob"): string | string[] | Promise<Blob[]>;
-    exportPCM(length?: number, accuracy?: number, noWindow?: boolean, start?: number, end?: number): Promise<string>;
+    exportPCM(length?: number, accuracy?: number, noWindow?: boolean, start?: number, end?: number): Promise<Peaks>;
```

Comment:
I would probably use an array here.

Reply:
The `Peaks` type is in fact just an an alias for an `Array`, as defined in the `index.d.ts`:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/25a17e04a81dd5bb9b97b3cec89d14af45926674/types/wavesurfer.js/index.d.ts#L123

The `ReadonlyArray<number> | ReadonlyArray<ReadonlyArray<number>>` part was there before me, but in different places, so I added the alias to avoid repetition and to semantically relate the occurrences..

According to https://wavesurfer-js.org/api/class/src/mediaelement.js~MediaElement.html#instance-method-getPeaks, `getPeaks()`, `Peaks` is either an Array of numbers, or an Array of Arrays of numbers. Is that still accurate? (Guess it makes using the return value rather hard, because users have to check what they got?)

Reply:
FYI I will open a follow-up PR later, fixing some issues as well as adding property docs and some missing elements (like `CanvasEntry.applyCanvasTransforms`, which seems to have been added in 5.0).